### ISSUE TITLE
Get channel view.

### ISSUE THREAD

**Issue by Alexminator99** [flyerhq/flutter_chat_ui#45]
I want to have the possibilites to remove the message input and replaced with a text like:
**Sender does not accept replies.**
Just like Google Messages.

**Code snippets 📝**
If applicable, add code samples to help explain your feature.

```dart
return Chat(
      messages: messages,
      isChannel: true // look here, this is what i want.
      onSendPressed: null, // this will be by default null if isChannel is true.
    );
  }
```

**Comment by demchenkoalex** [comment]
Hi @Alexminator99 please add screenshots and correct markdown. Why "Sender does not accept replies" is a header? Is it something you want to replace message input with?

**Comment by Alexminator99** [comment]
![Screenshot_20210614-193501__01.jpg](https://user-images.githubusercontent.com/51393761/121971858-bf40d000-cd47-11eb-9575-317de0b81326.jpg)

I manage to accomplish that by making a fork and then added the feature.

Now you understand what I meant?

**Comment by demchenkoalex** [comment]
I meant screenshot from Google Messages :) I can't remember a messenger where there is a message on the bottom when you look at the channel

**Comment by Alexminator99** [comment]
![Screenshot_20210614-200658.jpg](https://user-images.githubusercontent.com/51393761/121973825-1ba5ee80-cd4c-11eb-8272-c153cfcde825.jpg)

Look what I want to be a feature... Is very simple...

**Comment by jlubeck** [comment]
This seems to be similar as announcement channels, or even when admins MUTE a channel.

**Comment by Alexminator99** [comment]
> This seems to be similar as announcement channels, or even when admins MUTE a channel. Maybe instead of `isChannel` boolean, you can just put an optional String to change what text you want to put in the disabled input?

Sure... Whatever option that tells the user that is a channel and it can only read messages is valid...

**Comment by demchenkoalex** [comment]
Released in [v1.3.2](https://github.com/flyerhq/flutter_chat_ui/releases/tag/v1.3.2).
